airflow/config_templates/config.yml

```
@@ -2049,6 +2049,18 @@ scheduler:
       type: integer
       example: ~
       default: "60"
+    stale_dag_threshold:
+      description: |
+        How long (in seconds) to wait after we've reparsed a DAG file before deactivating stale
```

```suggestion
        How long (in seconds) to wait after we have reparsed a DAG file before deactivating stale
```

This should fix, spellchecker doesn't like some words with `'`

```console
Incorrect Spelling: 'reparsed'
Line with Error: 'How long (in seconds) to wait after we've reparsed a DAG file before deactivating stale'
Line Number: 6812
```

---

You could also check it in [Breeze](https://github.com/apache/airflow/blob/main/BREEZE.rst#building-the-documentation) locally